### PR TITLE
Add Icons for Github and Arrow and fix a couple whitespace issues on README and add another utility for Tailwind in globals.css

### LINKED ISSUE #25: Link to the repo from the site
To make it easier to navigate the project & get started contributing, there should be a link to the repo from the main site.

Do we need to update designs before this issue is ready to be picked up? Not sure what the workflows should look like with design (do we make a new label for "design needed" ?)

### PR DESCRIPTION
**Pull request by sarthaktexas**

### CODE REVIEW

**Review by exu3 — DISMISSED** [review]
Looks good to me. Thanks Sarthak!

File: README.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ We're building a community for discovery and learning.
 
 ## How do I contribute?
 Great question! You are expected to fork this repository and to create PRs when you want to merge your changes. Your commits are expected to follow our [Commit Convention](https://github.com/ThinkFiveable/open/blob/main/CONTRIBUTING.md#commiting), so some example ones are:
-- `fix: broken button on index.html`  
-- `fix(index): broken button`  
-- `chore: add NEWFILE.md`  
-- `feat: new button!!!`   
+- `fix: broken button on index.html`
+- `fix(index): broken button`
```
Comment (zaida04) [inline README.md]:
```suggestion
- `fix(Index): broken button`
```
can we use this PR to fix the commit message example issue that you brought up to me?

**Review by zaida04 — CHANGES_REQUESTED** [review]

**Comment by zaida04** [comment]
resolves #25

File: README.md (hunk 1 above)
Comment (sarthaktexas) [reply README.md]: sure

**Comment by awlevin** [comment]
#### 📝Changed routes:


| [**/**](https://open-i9vvskfhg-thinkfiveable.vercel.app/) |
|:-:|
|<a href="https://open-i9vvskfhg-thinkfiveable.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/RY8Jgszv9yo51C0vAlpF7RM2sws_.png">(view full size)</a> |







Commit 3b79690677bd0306e6aceafa01c745af34a7e666 (https://open-i9vvskfhg-thinkfiveable.vercel.app).

**Comment by sarthaktexas** [comment]
WAIT WE HAVE VERCEL COMMIT PREVIEW?

**Review by zaida04 — APPROVED** [review]

**Review by exu3 — APPROVED** [review]

**Pull request merged**